### PR TITLE
allow .min-wd file to be loaded as a node module

### PR DESCRIPTION
**Pull request by kolodny**

### CODE REVIEW

File: lib/options.js [hunk 1]
```diff
@@ -20,8 +21,14 @@ function override(options, overrides) {
 }
 
 function loadOptions(fileName, key, options) {
+  var overrides;
+
   if (fs.existsSync(fileName)) {
-    var overrides = JSON.parse(fs.readFileSync(fileName, 'utf8'));
+    try {
+      overrides = require(path.resolve(fileName));
```
Comment (mantoni) [inline lib/options.js]: Isn't this line enough? Requiring .json files works as well, so the catch-case is just trying to do the same thing again.

Comment (kolodny) [inline lib/options.js]: Nope, it needs to have a `.json` extension for that to work:

```bash
➜  ~  cd $(mktemp -d -t require)
➜  require.qcOFAP9n  echo '{"a": 123}' > a
➜  require.qcOFAP9n  echo 'console.log(require("./a"))' | node
/private/var/folders/_2/4d8dvsfx3_d6jmdmvtz_lv8c0000gn/T/require.qcOFAP9n/a:1
(function (exports, require, module, __filename, __dirname) { {"a": 123}
                                                                  ^

SyntaxError: Unexpected token :
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at [stdin]:1:13
    at Object.exports.runInThisContext (vm.js:54:17)
    at Object.<anonymous> ([stdin]-wrapper:6:22)
➜  require.qcOFAP9n  echo '{"b": 456}' > b.json
➜  require.qcOFAP9n  echo 'console.log(require("./b"))' | node
{ b: 456 }
```

Comment (mantoni) [inline lib/options.js]: Ah yes of cause. Makes sense :+1:

**Comment by mantoni** [comment]
Released in `2.9.0`.

**Pull request merged**